spec/helpers/checkouts_helper_spec.rb

```diff
@@ -51,20 +51,40 @@
 
     context 'when an item is not recalled' do
       it 'renders the right html' do
-        expect(content).to have_css('span', text: 'November 14, 2019 11:59pm')
+        expect(content).to have_css('span', text: 'November 14, 2019')
       end
     end
 
     context 'when an item is recalled' do
       before do
         allow(checkout).to receive_messages(
           recalled?: true,
-          recall_due_date: Time.zone.parse('2019-11-10T23:59:00-05:00')
+          recall_due_date: Time.zone.parse('2019-11-10T22:30:00-05:00')
         )
       end
 
       it 'renders the right html' do
-        expect(content).to include('<span>Recalled<br>November 10, 2019 11:59pm<br>November 14, 2019 11:59pm</span>')
+        expect(content).to include('<span>Recalled<br>November 10, 2019 10:30pm<br>November 14, 2019</span>')
+      end
+    end
+  end
+
+  describe '#format_due_date' do
+    let(:content) { helper.format_due_date(due_date) }
+
+    context 'when due date time is 11:59pm' do
+      let(:due_date) { Time.zone.parse('2019-11-10T23:59:00-05:00') }
+
+      it 'renders the right html' do
+        expect(content).to eq('November 10, 2019')
+      end
+    end
+
+    context 'when due date time is not 11:59pm' do
+      let(:due_date) { Time.zone.parse('2019-11-10T22:30:00-05:00') }
+
+      it 'renders the right html' do
+        expect(content).to include('November 10, 2019 10:30pm')
```

Comment:
```suggestion
        expect(content).to eq('November 10, 2019 10:30pm')
```

make it consistent with above spec